docker/bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Set some vars
+prepopvols=("config" "mods" "art") # these are folders which contain runtime needed files, and need to be represented in the host
```

Comment:
nit: UPPER_CASE_VARS to match other scripts in the repo

Reply:
done in latest commit